.idea/jarRepositories.xml

```diff
@@ -6,6 +6,16 @@
       <option name="name" value="Central Repository" />
       <option name="url" value="https://repo.maven.apache.org/maven2" />
     </remote-repository>
+    <remote-repository>
+      <option name="id" value="jitpack.io" />
+      <option name="name" value="jitpack.io" />
+      <option name="url" value="https://jitpack.io" />
+    </remote-repository>
+    <remote-repository>
+      <option name="id" value="github" />
+      <option name="name" value="github" />
+      <option name="url" value="https://github.com/ImNotStable/QualityEconomy" />
```

Comment:
Why did you hook it to your economy plugin?

Reply:
This is a mistake I'll figure out how to resolve this, I'm unsure of the exact cause.